lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -194,7 +197,7 @@ def test_todays_date_block(self):
         'info',
         'openedx.course_experience.course_home',
     )
-    @override_flag(UNIFIED_COURSE_EXPERIENCE_FLAG, active=True)
+    @override_flag_plus(course_experience_config(), UNIFIED_COURSE_EXPERIENCE_FLAG, active=True)
```

Comment:
Yes.  I think this would work if this were now an object, and would be cleaner to use.  I did not like what I current had.